### DISCUSSION TITLE
Coinmate as exchange rate provider

### DISCUSSION THREAD

**Discussion by prusnak** [btcpayserver/btcpayserver#4825] [Ideas / Feature Requests]
The URL is `https://coinmate.io/api/ticker?currencyPair={FROM}_{TO}`, examples:
* https://coinmate.io/api/ticker?currencyPair=BTC_CZK
* https://coinmate.io/api/ticker?currencyPair=BTC_EUR

**Comment by NicolasDorier** [comment]
Do they have an API which give all the rates on a single page instead?
Our abstraction is done in a way that a single request get all the tickers at once. Then the client of the abstraction decide which pair make sense for him in what is returned.

**Comment by prusnak** [comment]
https://coinmate.io/api/tickerAll
